src/components/private/onsite-payment.tsx

```
@@ -0,0 +1,128 @@
+                </Select>
+                <Button
+                  onClick={handleConfirm}
+                  disabled={!amount || !paymentOption}
+                  className="w-full bg-emerald-500 text-white hover:bg-emerald-600"
+                >
+                  Confirm
+                </Button>
+              </div>
+            </div>
+          </>
+        )}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)